src/css/main.scss

```diff
@@ -0,0 +1,94 @@
+@import "../css/body";
+body {
+  background-color: #f1f1f1;
+}
+
+.container {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 0 20px;
+  .name {
+    font-size: 40px;
+    font-family: Georgia, "Times New Roman", Times, serif;
+    text-align: center;
+  }
+  p {
+    text-align: center;
+    font-family: Georgia, "Times New Roman", Times, serif;
+  }
+  .text-wrapper {
+    width: 840px;
+    margin: auto;
+  }
+  textarea {
+    outline: none;
+    width: 820px;
+    padding: 10px;
+    border-radius: 10px;
+    border: 2px solid rgb(141, 134, 134);
+    font-size: 20px;
+    font-family: Georgia, "Times New Roman", Times, serif;
+    background-color: #f1f1f1;
+  }
+}
+
+.keyboard-list {
+  padding: 10px;
+  width: 860px;
+  margin: auto;
+  display: flex;
```

Comment:
Please learn about possible approaches in css properties order. I personally prefer [this one](http://codeguide.academy/html-css.html#css-order), but you can choose the one you like. Just follow some and don't leave it random.

Reply:
Made according this rule